setup.cfg

```
@@ -42,7 +42,7 @@ install_requires =
     environs >= 8.0.0
     opentelemetry-api >= 0.11b0
     opentelemetry-sdk >= 0.11b0
-    opentelemetry-ext-otlp >= 0.11b0
+    opentelemetry-exporter-otlp >= 0.11b0
```

```suggestion
    opentelemetry-exporter-otlp >= 0.12b0
```
this needs the new release of opentelemetry-python